src/main/resources/plugin.rnc

```diff
@@ -59,8 +60,10 @@ extension-point =
 require =
   element require {
     attribute plugin { xsd:NCName },
+    attribute version { version },
```

Comment:
Does this mean `@version` will now be required on the `<require>` element? Might be better to make it optional (default to `0.0.0` which is the default for plugins without a version)... otherwise we'll make a lot of existing plugins invalid.